src/fidesops/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -133,8 +136,57 @@ def consent_request_verify(
     return _prepare_consent_preferences(db, provided_identity)
 
 
-@router.patch(
+@router.post(
     CONSENT_REQUEST_PREFERENCES,
+    dependencies=[Security(verify_oauth_client, scopes=[CONSENT_READ])],
+    status_code=HTTP_200_OK,
+    response_model=ConsentPreferences,
+)
+def get_consent_preferences(
+    *, db: Session = Depends(get_db), data: Identity
+) -> ConsentPreferences:
+    """Gets the consent preferences for the specified user."""
+    if data.email:
+        lookup = data.email
+    elif data.phone_number:
+        lookup = data.phone_number
+    else:
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST, detail="No identity information provided"
+        )
+
+    identity = ProvidedIdentity.filter(
+        db,
+        conditions=(
+            (ProvidedIdentity.hashed_value == ProvidedIdentity.hash_value(lookup))
+            & (
+                ProvidedIdentity.privacy_request  # pylint: disable=singleton-comparison
+                == None
```

Comment:
I'd use the `ProvidedIdentity.privacy_request_id.is_(None)` filter here instead of filtering on a field that's not actually a field and having to add the pylint ignore

Reply:
I still don't understand why, but I had some bad merge conflicts when merging main into this and it looks like I messed up the resolution in some places (all the below look to be involved). Fixing now.